src/core.js

```diff
@@ -77,8 +66,19 @@ extend( QUnit, {
 
 	},
 
-	// DEPRECATED: QUnit.asyncTest() will be removed in QUnit 2.0.
-	asyncTest: asyncTest,
+	asyncTest: function() {
+		throw new Error(
+			"asyntTest is now removed from QUnit 2.0, check out our upgrade guide at " +
```

Comment:
I've set throws only for removed methods as asyncTest, init, stop, and the previously exposed globals.

I skipped throwing for features as setup/teardown, test signature, etc.